mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule GrpcReflection.MixProject do
   use Mix.Project
 
-  @version "0.1.0"
+  @version "0.2.0"
```

Comment:
This PR makes zero functional changes to the library, only changing test structures and a dependency in the example project.  It should increment the patch number instead of the minor version number

```suggestion
  @version "0.1.1"
```